x-pack/plugins/cases/public/common/user_actions/parsers.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  CaseUserActionConnectorRt,
+  CaseConnector,
+  ConnectorTypes,
+  noneConnectorId,
+  CaseFullExternalService,
+  CaseUserActionExternalServiceRt,
+} from '../../../common';
+
+export const parseStringAsConnector = (
+  id: string | null,
+  encodedData: string | null
+): CaseConnector | undefined => {
```

Comment:
nit: I think is best to return `null` to intentionally represent no value (no connector).